docs/modules/data-structures/pages/ringbuffer.adoc

```diff
@@ -295,9 +295,9 @@ hazelcast:
 ----
 ====
 
-The following are the explanations for the Ringbuffer store configuration elements:
+The following are the configuration elements and their descriptions:
```

Comment:
The configuration elements are as follows:

Reply:
@rebekah-lawrence I don't mind this change, but I used the same wording as in [List](https://docs.hazelcast.com/hazelcast/latest/data-structures/list#configuring-list), [Set](https://docs.hazelcast.com/hazelcast/latest/data-structures/set#configuring-set) & [MultiMap](https://docs.hazelcast.com/hazelcast/latest/data-structures/multimap#configuring-multimap) - so maybe they should all be changed separately?

Reply:
If it matches the other sections, just leave it as-is. Consistency is king ;-)
We can re-visit later when reassessing the docs as a whole.
Thanks!